### PR TITLE
Periodic_3_triangulation_3: Some accelerations

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Add timers in all tests, remove an old expensive assertion, and repla…ce a map with an unordered_map

## Release Management

* Affected package(s): Periodic_3_triangulation_3

### CODE REVIEW

File: Periodic_3_triangulation_3/include/CGAL/Periodic_3_Delaunay_triangulation_3.h [hunk 1]
```diff
@@ -370,11 +370,10 @@ class Periodic_3_Delaunay_triangulation_3 :
             edge_to_delete = std::make_pair((*it)->vertex(k),(*it)->vertex(j));
           }
           Vertex_handle v_no = edge_to_delete.first;
-          sit = std::find(too_long_edges[v_no].begin(),
-              too_long_edges[v_no].end(),
-              edge_to_delete.second);
-          if(sit != too_long_edges[v_no].end()) {
-            too_long_edges[v_no].erase(sit);
+		  std::list<Vertex_handle>& vl = too_long_edges[v_no];
```
Comment (MaelRL) [inline Periodic_3_triangulation_3/include/CGAL/Periodic_3_Delaunay_triangulation_3.h]: indentation

File: Periodic_3_triangulation_3/include/CGAL/Periodic_3_Delaunay_triangulation_3.h [hunk 2]
```diff
@@ -408,12 +407,12 @@ class Periodic_3_Delaunay_triangulation_3 :
           p1 = construct_point((*it)->vertex(j)->point(), get_offset(*it, j));
           p2 = construct_point((*it)->vertex(k)->point(), get_offset(*it, k));
 
+		  std::list<Vertex_handle>& vl = too_long_edges[(*it)->vertex(j)];
```
Comment (MaelRL) [inline Periodic_3_triangulation_3/include/CGAL/Periodic_3_Delaunay_triangulation_3.h]: same

**Comment by lrineau** [comment]
Do we want that patch in CGAL-4.11.1?

File: Periodic_3_triangulation_3/include/CGAL/Periodic_3_triangulation_3.h [hunk 3]
```diff
@@ -181,7 +181,7 @@ class Periodic_3_triangulation_3
 private:
   typedef typename GT::FT                      FT;
   typedef std::pair< Vertex_handle, Offset >   Virtual_vertex;
-  typedef std::map<Vertex_handle, Virtual_vertex>
+  typedef boost::unordered_map<Vertex_handle, Virtual_vertex>
```
Comment (MaelRL) [inline Periodic_3_triangulation_3/include/CGAL/Periodic_3_triangulation_3.h]: Change the reverse map for symmetry ?

**Comment by afabri** [comment]
I don't see a need for old releases

File: Periodic_3_triangulation_3/include/CGAL/Periodic_3_triangulation_3.h (hunk 3 above)
Comment (afabri) [reply Periodic_3_triangulation_3/include/CGAL/Periodic_3_triangulation_3.h]: done

**Comment by lrineau** [comment]
Tested in [CGAL-4.12-Ic-152], successfully, and then merged.

  [CGAL-4.12-Ic-152]: https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-152.shtml

**Pull request merged**